git_machete/generated_docs.py

```diff
@@ -0,0 +1,944 @@
+
+        Every branch name can be followed (after a single space as a delimiter) by a custom annotation - a PR number in the above example.
+        The annotations don't influence the way `git machete` operates other than that they are displayed in the output of the `status` command.
+        Also see anno command.
+
+        Tabs or any number of spaces can be used as indentation.
+        It's only important to be consistent wrt. the sequence of characters used for indentation between all lines.
+   """,
+    "github": """
+        <b>Usage:</b><b>
+          git machete github <subcommand></b>
+
+        where `<subcommand>` is one of: `anno-prs`, `checkout-prs`, `create-pr`, `retarget-pr`.
+
+        Creates, checks out and manages GitHub PRs while keeping them reflected in branch definition file.
+
+        To allow GitHub API access for private repositories (and also to perform side-effecting actions like opening a PR, even in case of public repositories),
+        a GitHub API token with `repo` scope is required, see https://github.com/settings/tokens. This will be resolved from the first of:
+        * `GITHUB_TOKEN` env var,
+        * content of the `.github-token` file in the home directory (`~`),
+        * current auth token from the `gh` GitHub CLI,
+        * current auth token from the `hub` GitHub CLI.
+
+        GitHub API server URL will be inferred from `git remote`.
+        You can override this by setting the following git config keys:
+           Remote name
+              E.g. `machete.github.remote` = `origin`
+
+           Organization name
+              E.g. `machete.github.organization` = `VirtusLab`
+
+           Repository name
+              E.g. `machete.github.repository` = `git-machete`
+
+        To do this, run `git config --local --edit` and add the following section:
+        <dim>
+          [machete "github"]
+              organization = <organization_name>
+              repository = <repo_name>
+              remote = <remote_name>
+        </dim>
+
+        `anno-prs`:
+        Annotates the branches based on their corresponding GitHub PR numbers and authors.
+        Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
+        Equivalent to `git machete anno --sync-github-prs`.
+
+        `checkout-prs [--all | --by=<github-login> | --mine | <PR-number-1> ... <PR-number-N>]`:
+
+        Check out the head branch of the given pull requests (specified by numbers or by a flag),
+        also traverse chain of pull requests upwards, adding branches one by one to git-machete and check them out locally.
+        Once the specified pull requests are checked out locally, annotate local branches with corresponding pull request numbers.
+        If only one PR has been checked out, then switch the local repository's HEAD to its head branch.
+
+        <b>Options:</b>
```

Comment:
Huh would it be doable to indent the `Options` for subcommands? or best, indent the entire description for each subcommand

Reply:
As it's pretty unclear now where one subcommand ends and another begins ☹️

Reply:
@PawelLipski Yes, it's done, also i made some changes to the `github` .rst file